src/pages/Thanks/ThanksPage.vue

```diff
@@ -302,6 +303,14 @@ export default {
 		userHasLentBefore(hasLentBefore);
 		userHasDepositBefore(hasDepositBefore);
 
+		// MARS-246 Hotjar user attributes
+		setHotJarUserAttributes({
+			userId: data?.my?.userAccountId?.id,
+			hasEverLoggedIn: true,
```

Comment:
hasEverLoggedIn isn't necessarily true here. The user may have used guest checkout

Reply:
CampaignThanks is currently only shown after a successful checkout in that context which can only occur after being logged in. Guest Checkout isn't available here so it's prob ok for now to remain set to `true`

Reply:
Agreed that CampaignThanks can just be set to `true`. However, for this line in ThanksPage I think it should be changed to `hasEverLoggedIn: isLoggedIn`.